src/prefect/orion/schemas/actions.py

```diff
@@ -109,6 +111,37 @@ class DeploymentCreate(ActionBaseModel):
     entrypoint: Optional[str] = FieldFrom(schemas.core.Deployment)
     infra_overrides: Optional[Dict[str, Any]] = FieldFrom(schemas.core.Deployment)
 
+    @staticmethod
```

Comment:
All of this junk is duplicated in both `create` and `update`. I originally had it in `schemas.core.Deployment` but then that has me calling `schemas.core.Deployment.check_valid_configuration` which felt awkward too. I also thought about making these standalone functions, but `actions` is already pretty cluttered as-is. 

I'm like some suggestions here.